### PR TITLE
Chore/update deps

### LINKED ISSUE #39: [Task]: Upgrade dependencies

### PR DESCRIPTION
**Pull request by aspnxdd**
Fixes #39

## Changes

  - Update deps
  
### Checks

* [x] Have you catchup your branch with the latest state in base?
* [x] Have you lint your code locally prior to submission?
* [x] Have you reviewed your proposed changes and removed debris?
* [x] Have you successfully ran due tests with your changes?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,24 +33,24 @@
...
     "eslint": "^8.32.0",
-    "tsup": "^6.5.0",
-    "typescript": "^4.7.4"
+    "typescript": "^4.9.4",
+    "tsup": "^6.5.0"
   },
   "dependencies": {
     "@babel/core": "^7.20.12",
     "@babel/eslint-parser": "^7.19.1",
     "@metaplex/arweave-cost": "^1.0.4",
```
Comment (evavirseda) [inline package.json:45]: I think we should also update `@metaplex/arweave-cost`
image

**Review by evavirseda — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (aspnxdd) [reply package.json:45]: @evavirseda thing is i did it on purpose. Last time I tried 2.0.0 it had an error, in which the cost was lower than it should, so I could not upload anything to arweave at all :|

Comment (evavirseda) [reply package.json:45]: that's great, then :ok_hand:

**Review by evavirseda — APPROVED** [review]
:rocket:

**Pull request merged**